src/TasksModule/Components/TasksForm/TasksForm.tsx

```diff
@@ -0,0 +1,227 @@
+    }
+  };
+
+  const getusersList = async () => {
+    try {
+      const response = await axios.get(`${baseUrl}/Users`, {
+        headers: { Authorization: token },
+      });
+      console.log("users list ", response.data.data);
+      setUsersList(response.data.data);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {
+    getusersList();
+    if (action == "update") getCurrentTask();
+    else getprojectsList();
+  }, []);
+
+  return (
+    <>
+      <div className="TaskFormContainer vh-100 bg-light overflow-auto pageOverflow pb-2">
+        <div className="TaskFormHeader bg-white ps-5 py-4">
+          <a href="/dashboard/tasks-list" className="text-black">
+            <i className="fa fa-arrow-left me-2"></i>
+            View All Tasks
+          </a>
+          <h2 className="mt-2">
+            {action == "add" ? "Add a New Task" : "Update Task"}
+          </h2>
+        </div>
+        <form
+          onSubmit={handleSubmit(onSubmit)}
+          className={`${styles.TaskForm} rounded-4 bg-white shadow container  w-75`}
+        >
+          <div className={`mt-5 py-5 px-4 TaskFormInputs`}>
+            <div className="title ">
+              <label htmlFor="">Title</label>
+              <input
+                type="text"
+                placeholder="Name"
+                className="form-control rounded-4 p-3 mt-2"
+                {...register(
+                  "title",
+                  action == "add" ? { required: "Task title is required" } : {}
+                )}
+              />
+            </div>
+            {errors.description && (
+              <p className="text-danger">{errors?.title?.message}</p>
+            )}
+
+            <div className="description mt-4 ">
+              <label htmlFor="">Description</label>
+              <textarea
+                placeholder="Description"
+                className="form-control rounded-4 mt-2 pt-3 ps-3 pe-1 pb-5"
+                {...register(
+                  "description",
+                  action == "add"
+                    ? { required: "Task description is required" }
+                    : {}
+                )}
+              ></textarea>
+            </div>
+            {errors.description && (
+              <p className="text-danger">{errors?.description?.message}</p>
+            )}
+
+            <div className="options row mt-4">
+              <div className={action == "add" ? "col-md-6" : "col-md-12"}>
+                <label htmlFor="">User</label>
+                <select
+                  name="user"
+                  id=""
+                  className="w-100 form-select mt-2 rounded-4 p-3"
+                  {...register("employeeId", {
+                    required: action == "add" ? "User is required" : false,
+                  })}
+                >
+                  {action == "add" ? (
+                    <option value="" disabled selected hidden>
+                      No Users Selected
```

Comment:
I suggest making it as `select an employee` which I feel is a little bit more friendly than just writing `No Users Selected` and it also guides the user on what to do.